src/Pages/PayPage/Pay.scss

```
@@ -0,0 +1,56 @@
+@import '../../styles/common.scss';
+
+.pay {
+  max-width: 900px;
+  margin: 0 auto;
+  border: $border;
+  color: $black;
+  .header {
```

```suggestion
  color: $black;
  
  .header {
```
이것도 앞서 드렸던 리뷰인거 같은데 새로운 셀렉터 앞에서는 한줄 띄워주시는게 좋습니다

---

아.. 일부러 다 붙였는데, 감사합니다 바로 적용하겠습니다!